fabtests/functional/rdm_tagged_peek.c

```diff
@@ -50,9 +50,12 @@ static int wait_for_send_comp(int count)
 	struct fi_cq_tagged_entry comp;
 
 	do {
-		ret = fi_cq_sread(txcq, &comp, 1, NULL, -1);
+		ret = fi_cq_read(txcq, &comp, 1);
```

Comment:
Hmm, overlooked the FT_COMP_SREAD option, even though it might be just an arbitrary choice.  For now, we can't drop this commit.  Later the affected provider cab ne improved to make progress inside sread().

Reply:
commit dropped.

Reply:
I do think SREAD was an arbitrary choice.  I can confirm in practice that MPI spins on peek operations until they succeed.